tests/functional/test_title.py

```diff
@@ -10,3 +10,10 @@ def test_contact_page(selenium):
     selenium.find_element_by_xpath("//body//a[text()='Contacts']").click()
     body_text = selenium.find_element_by_tag_name("body").text
     assert "CONTACT" in body_text, "'CONTACT' string wasn't found in body text"
+
+
+def test_cycles_page(selenium):
+    selenium.get('http://localhost:5000')
+    selenium.find_element_by_xpath("//body//a[text()='Cycles']").click()
+    body_text = selenium.find_element_by_tag_name("body").text
+    assert "Sessions" in body_text, "'Sessions' string wasn't found in body text of the 'cycles' page"
```

Comment:
It is better to check the resulting url after the click. The page content may change which will break the test.

Reply:
I disagree. We always need to check the web page content. Because if we only check the URL, we might miss the cases when the content couldn't be rendered for some reason. This is a common practice to check the web page content in order to decide whether it was loaded or not. Selenium automation is such that it WILL brake in case of html change. I am taking this in consideration.

Reply:
It still doesn't cover the case where you got into the wrong end-point.  You might want to add another check.

Reply:
If the user clicks on 'cycles' tab on home page, the user is expected to be navigated to a page with a particular content.
The way check if the content was rendered successfully should be sufficient test.
If the user will be navigated by mistake to some other end-point, the test will fail because the content wont be found on the page.

Reply:
> If the user clicks on 'cycles' tab on home page, the user is expected to be navigated to a page with a particular content.
> The way check if the content was rendered successfully should be sufficient test.
> If the user will be navigated by mistake to some other end-point, the test will fail because the content wont be found on the page.
Maybe and maybe not. 
Is there a problem in implementing the url check?

Reply:
Implemented the URL check, and changed the 'cycles' page content.
See here:
https://image.op.redhat.com/view/MiC-dTyMg